### PR TITLE
updated confluent readme for input files & removed phone number property

### PR DESCRIPTION
**Pull request by ankisho**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ankisho** [comment]
@erich-wang : Could you please review this PR ?

**Comment by erich-wang** [comment]
@ankisho , the lable `WaitForARMFeedbak` was added to ask arm review board to review. Adding pattern to existing parameter is kind of break change for customers, has the change been deployed to production?

**Comment by ankisho** [comment]
> @ankisho , the lable `WaitForARMFeedbak` was added to ask arm review board to review. Adding pattern to existing parameter is kind of break change for customers, has the change been deployed to production?

This PR has changes related to removing phone number property. This is breaking change.
right now, this RP is on production but it is not come under private preview yet. This RP is not being used by any of azure end-user. I'm doing internal testing using this RP and Private preview is scheduled for 31st July 2020. I've one more PR with similar changeswith RPaaS team: [https://github.com/Azure/azure-rest-api-specs-pr/pull/1449](url)

**Comment by ankisho** [comment]
@erich-wang : Could you please review this PR?

**Comment by ankisho** [comment]
Could you please help me to review this PR? @erich-wang

**Review by chiragg4u — CHANGES_REQUESTED** [review]
There are breaking changes in this PR. Please take one of the options
1.	confirm if there is no customers for this version
2.	update the API version
3.	follow Azure Breaking changes approval request form as defined @ [Breaking Change Process](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37673/Retirements-and-Breaking-Changes)

File: specification/confluent/resource-manager/Microsoft.Confluent/preview/2020-03-01-preview/confluent.json [hunk 1]
```diff
@@ -538,8 +538,7 @@
           "type": "string"
         },
         "phoneNumber": {
-          "description": "Phone number. Validation Pattern: ^(\\+\\d{1,2}\\s)?\\(?\\d{3}\\)?[\\s.-]?\\d{3}[\\s.-]?\\d{4}$",
-          "pattern": "^(\\+\\d{1,2}\\s)?\\(?\\d{3}\\)?[\\s.-]?\\d{3}[\\s.-]?\\d{4}$",
```
[the remark this answers is not in the document]
Comment (ankisho) [reply specification/confluent/resource-manager/Microsoft.Confluent/preview/2020-03-01-preview/confluent.json:542]: This change is required as we had discussion with ISV Confluent. We & confluent ISV are decided that phone number is not required for this RP and requested to remove this.

**Comment by ankisho** [comment]
> There are breaking changes in this PR. Please take one of the options
> 
> 1. confirm if there is no customers for this version
> 2. update the API version
> 3. follow Azure Breaking changes approval request form as defined @ [Breaking Change Process](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37673/Retirements-and-Breaking-Changes)

This PR has changes related to removing phone number property. This is breaking change.
right now, this RP is in production behind hidden feature flag but it does not come under private preview yet. This RP is not being used by any of azure end-user. I'm doing internal testing using this RP and Private preview is scheduled for 31st July 2020. 
I've one more PR with similar changes with RPaaS team: https://github.com/Azure/azure-rest-api-specs-pr/pull/1449

@chiragg4u

**Review by chiragg4u — APPROVED** [review]
Approving based as the feature is under testing.

**Comment by ankisho** [comment]
> Approving based as the feature is under testing.

thanks

**Comment by ankisho** [comment]
@erich-wang : Could you please prioritize the merge of this PR? This is blocking the ARM template generation task.

**Review by erich-wang — APPROVED** [review]

**Comment by erich-wang** [comment]
@akning-ms, could you please force merge this PR?

**Comment by ankisho** [comment]
> @akning-ms, could you please force merge this PR?

@erich-wang @akning-ms : I have a question about this swagger json. we are currently on private preview stage of RP development. since this repo is a public repo, should we merge this swagger to public repo before public private stage?

**Comment by akning-ms** [comment]
Hi @ankisho, Normally, for private review. you can just submit PR in azure-rest-api-specs-pr(private repo). but looks your preview version 2020-03-01-preview was already in public repo, this PR is just kind of fix. so you can continue your merge for this version if you want. so let us know your thoughts

**Comment by ankisho** [comment]
> Hi @ankisho, Normally, for private review. you can just submit PR in azure-rest-api-specs-pr(private repo). but looks your preview version 2020-03-01-preview was already in public repo, this PR is just kind of fix. so you can continue your merge for this version if you want. so let us know your thoughts

Hi @akning-ms , i've some updates regarding this. 
since, confluent RP is going for private preview phase now(july 2020 end). we have decided that it'd be good if we can expose the confluent RP schema spec to public from Public preview or GA phase. Hence, i've updated this PR to remove the confluent spec from this public repo. Kindly review.

**Pull request merged**